src/reducers/scrollStateHelper.js

```diff
@@ -347,6 +353,7 @@ export function scrollToRow(state, rowIndex) {
     var position = _getRowAtEndPosition(state, rowIndex);
     return scrollTo(state, position);
   }
+  // TODO (jordan) Should this be a no-op?
```

Comment:
This is where someone asks us to scroll to a row and it's already in the viewport.  Seems like we could just return / return 0 instead...